bedrock/products/templates/products/vpn/platforms/v2/linux.html

```diff
@@ -44,7 +44,13 @@ <h3>{{ ftl('vpn-shared-platform-what-is') }}</h3>
 <p>{{ ftl('vpn-linux-mozilla-vpn-is-a') }}</p>
 <h3>{{ ftl('vpn-linux-why-choose') }}</h3>
 <ul class="mzp-u-list-styled">
-  <li>{{ ftl('vpn-linux-fast-and') }}</li>
+  <li>
+    {% if ftl_has_messages('vpn-linux-fast-and-v2') %}
+      {{ ftl('vpn-linux-fast-and-v2') }}
+    {% else %}
+      {{ ftl('vpn-linux-fast-and') }}
+    {% endif %}
```

Comment:
Nit: you can simplify this to

```suggestion
    {{ ftl('vpn-linux-fast-and-v2', fallback='vpn-linux-fast-and') }}
```